packages/compiler/src/core/compiler-code-fixes/show-linter-rule-doc-url.codefix.ts

```diff
@@ -0,0 +1,10 @@
+import { defineCodeFix } from "../diagnostics.js";
+
+export function createShowLinterRuleDocUrlCodeFix(docUrl: string) {
```

Comment:
This shouldn't be a code fix. It doesn't actually fix anything. It would break a --fix-all flag. This is only relevant to ide and so just need to be injected when creating the Vscode actions.
We are already including the doc url in many other place this is just another.